src/app/map/suitability-map-panel/suitability-map-panel.component.html

```diff
@@ -48,7 +49,7 @@ <h2 class="map__control__title title">
                         [checked]="level.checked"
                         [ngModel]="level.checked"
                         (ngModelChange)="onToggleCheckbox($event, level)">
-                        {{ level.name }}
+                        {{ 'LEVELS.' + level.translationKey | translate}}
```

Comment:
Add space after the translate pipe like this `{{ 'LEVELS.' + level.translationKey | translate }}`